rts/lra/lra-test/tck/pom.xml

```diff
@@ -21,7 +21,7 @@
         <lra.tck.suite.thorntail.trace.params></lra.tck.suite.thorntail.trace.params>
         <lra.tck.suite.debug.params></lra.tck.suite.debug.params> <!-- has content when -Ddebug.tck is specified -->
         <lra.tck.suite.debug.port>8788</lra.tck.suite.debug.port>
-        <lra.tck.consistency.delay>0</lra.tck.consistency.delay>
+        <lra.tck.consistency.delay>20000</lra.tck.consistency.delay>
```

Comment:
What is the reason of adding here this delay? It's little big high number, isn't it?

Reply:
The delay has to be big enough to allow the next periodic recovery pass to happen. I can reduce the periodic recovery period and this delay but the full suite seems to takes 10-20 minutes which I think is fine (but not ideal) since it use to only take a few minutes prior to the new tests introduced by Rudy.

Note that we are already setting the recovery period quite low and setting it too low is not advisable.

Reply:
Also note that we can set the delay on just Rudy's tests after https://github.com/eclipse/microprofile-lra/pull/219 is merged.

Reply:
ok, but the testsuite running 15 minutes instead of 2 mins sounds me rather as critical. I personally don't think this being a minor. Especially when some timeout needs to be setup to "a big number". This sounds rather as hiding some race condition functionality issue, doesn't it?

I confess that I do not follow the work on LRA spec in details so I could be missing something. But I really don't like fixing things with prolonging a timeout.

Reply:
There is no race condition, we are setting the delay long enough for "eventual consistency" to kick in. In theory the delay could be set to an arbitrarily large value since that would conform to the definition of eventual consistency.

You do not need to know the details of the spec, you just need to understand that the only guarantee provided by LRA is that the system will eventually become consistent. The question then is, in practice, how long is it reasonable to wait for consistency before failing the TCK test.

Reply:
Ok, I see and I do understand. No objection then :-)